android/BOINC/app/src/main/java/edu/berkeley/boinc/PrefsFragment.java

```diff
@@ -157,6 +157,7 @@ private void populateLayout() throws RemoteException{
 		Boolean stationaryDeviceMode = BOINCActivity.monitor.getStationaryDeviceMode();
 		Boolean stationaryDeviceSuspected = BOINCActivity.monitor.isStationaryDeviceSuspected();
 
+		// The order is important, the GUI will be displayed in the same order as the data is added.
```

Comment:
Tip for next time: always make sure your commits are atomic in order to be able to revert them if needed (like e62974a6834cfa55dc6c3ca275366ac22139d2ee in this case).